<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_shared_image</class>
 <widget class="QDialog" name="view_shared_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Request more views</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_views_left">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>59</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of views left: </string>
   </property>
  </widget>
  <widget class="QLabel" name="myImg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
